Mlem/Views/Tabs/Feed Root.swift

```diff
@@ -79,14 +71,22 @@ struct FeedRoot: View {
             } label: {
                 Text("Close")
             }
-            
+
         } message: {
             Text(appState.alertMessage)
         }
         .onAppear {
-            print("Saved thing from keychain: \(String(describing: AppConstants.keychain["test"]))")
-            if appState.currentActiveAccount == nil, let account = accountsTracker.savedAccounts.first {
-                appState.currentActiveAccount = account
+            if rootDetails == nil || shortcutItemToProcess != nil {
+                let feedType = FeedType(rawValue:
+                    shortcutItemToProcess?.type ??
+                    "nothing to see here"
+                ) ?? defaultFeed
+                var dits: CommunityLinkWithContext?
+                if appState.currentActiveAccount != nil {
+                    dits = CommunityLinkWithContext(community: nil, feedType: feedType)
```

Comment:
what is `dits`? is there a more descriptive name this variable could have?

Reply:
Addressed